alacritty/src/string.rs

```diff
@@ -50,6 +50,8 @@ impl<'a> StrShortener<'a> {
         }
 
         if direction == ShortenDirection::Right {
+            // clippy::iter_skip_zero appears in Rust 1.73, ignore it for earlier versions.
+            #[allow(unknown_lints, clippy::iter_skip_zero)]
```

Comment:
Oldstable clippy is the only thing that matters. If there's an issue fixing before oldstable starts complaining that's fine, but exceptions shouldn't be added before oldstable requires it.

Reply:
OK, I'm removing that change. I'll try to find an acceptable solution, but I don't see any good approach at the moment.